src/Abstractions/NexusMods.Abstractions.Games.Diagnostics/DiagnosticTemplateBuilder.cs

```diff
@@ -71,6 +96,11 @@ public interface IMessageBuilder
         /// Adds data references to the message.
         /// </summary>
         IMessageBuilder AddDataReference<T>(string name) where T : IDataReference;
+
+        /// <summary>
+        /// Adds a simple value to the message.
+        /// </summary>
+        IMessageBuilder AddValue<T>(string name) where T : notnull;
```

Comment:
I'm not really clear on what the use cases for `AddValues` are supposed to be. 
Is this to later support custom handlers? I would expect T to implement some kind of interface TBD in that case, or is this for something else?

Reply:
This is to add any kind of data you want to the diagnostic. This can be a `string`, `int`, or whatever makes sense in the context of the diagnostic.

Reply:
Is the only purpose of that data is for displaying in the description?

Reply:
For now, yes.